demo/admin/src/pages/blocks/FullWidthImageBlock.tsx

```diff
@@ -12,7 +12,7 @@ const FullWidthImageContentBlock = createOptionalBlock(RichTextBlock, {
 export const FullWidthImageBlock = createCompositeBlock({
     name: "FullWidthImage",
     displayName: <FormattedMessage id="blocks.fullWidthImage" defaultMessage="Full Width Image" />,
-    category: BlockCategory.Media,
+    category: { label: <FormattedMessage id="blocks.category.custom" defaultMessage="Custom" />, insertBefore: BlockCategory.Media },
```

Comment:
Although it doesn't make a difference, I would show the pattern to define the object centrally and use it in (possibly multiple) blocks. And maybe add a required `id` to avoid processing the label?

Reply:
> Although it doesn't make a difference, I would show the pattern to define the object centrally and use it in (possibly multiple) blocks. 

Done: 0345a48af244232a0f0d2f46969ad5713b4c456e

> And maybe add a required id to avoid processing the label?

Done: 436dd233e4872023fa7252e382d8135e4c89efc8

Reply:
> Although it doesn't make a difference, I would show the pattern to define the object centrally and use it in (possibly multiple) blocks.

:+1: can you also show that pattern in the changelog?

Reply:
Done: 7c484c4ff4bfe619b5435941c46e6dd2dddb7ba4